_sass/components/tab.scss

```diff
@@ -2,7 +2,7 @@
 //------------------------------------------------
 //------------------------------------------------
 
-// dynamic tab switching based on https: //levelup.gitconnected.com/tabbed-interfaces-without-javascript-661bab1eaec8
```

Comment:
a random space broke the URL -- this was caught by a different heuristic -- I'm not actively looking for it, but this does appear to be the only instance.